### PR TITLE
AppVeyor - collect code coverage

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

**Comment by keradus** [comment]
refs:
- https://github.com/satooshi/php-coveralls/issues/171
- https://github.com/satooshi/php-coveralls/issues/172

File: appveyor.yml [hunk 1]
```diff
@@ -12,18 +12,30 @@ cache:
...
 
 before_test:
     - cd C:\projects\php-cs-fixer
-    - php C:\tools\composer.phar update --prefer-source --ansi --no-interaction --no-dev
+    - php C:\tools\composer.phar update --no-interaction
 
 test_script:
     - cd C:\projects\php-cs-fixer
-    - php C:\tools\phpunit.phar --verbose
+    - php -c C:\tools\php\php-with_xdebug.ini C:\tools\phpunit.phar --verbose --coverage-clover build\logs\clover.xml
```
Comment (stof) [inline appveyor.yml]: do we really need to generate code coverage there too ?

Comment (GrahamCampbell) [inline appveyor.yml]: well, by that argument, we should only generate it on one php version on travis too

Comment (GrahamCampbell) [inline appveyor.yml]: the idea here is surely that execution paths on different php versions and OSs might not be the same

Comment (keradus) [inline appveyor.yml]: We already had some cases that works on unix but not on Windows, and as it great to fix them and test on Windows too, the fix may generate code that is reached only on Windows. So if possible - the code coverage should be collected as well.
For now PHP Coveralls won't work under Windows.

Comment (GrahamCampbell) [inline appveyor.yml]: > For now PHP Coveralls won't work under Windows.

lol, windows

Comment (keradus) [inline appveyor.yml]: No. The problem is PHP Coveralls.
It everything uses `/` in checks reading paths from disc, so all path-related checks fail

Comment (GrahamCampbell) [inline appveyor.yml]: The best sollution there is just call `realpath` rather than messing with the dir separator constant.

Comment (keradus) [inline appveyor.yml]: whatever solution - goto PHP Coveralls ;)

**Comment by keradus** [comment]
Closing, as should be done with phpdbg instead of xdebug

**Pull request closed without merging**